src/schema/rules/sidecars/pet.yaml

```diff
@@ -28,6 +28,8 @@ PETScannerHardware:
     BodyPart:
       level: recommended
       description_addendum: Corresponds to DICOM Tag 0018, 0015 `Body Part Examined`.
+    BodyPartDetails: optional
+    BodyPartDetailsOntology: optional
```

Comment:
Not sure how much sense it makes to add those to PET: @melanieganz @mnoergaard

I am only adding them now for internal consistency.